app/src/main/res/values-it/help.xml

```diff
@@ -51,32 +51,32 @@
         <h2>Seed Mnemonico</h2>
         Questa è l&apos;unica informazione necessaria per recuperare il tuo portafoglio in futuro
 		e avere pieno accesso ai tuoi fondi.
 		E&apos; molto importante mantenerlo privato e conservarlo in un luogo sicuro, poiché può concedere a <em>chiunque</em>
 		pieno accesso ai tuoi Moneroj! Se non lo hai già fatto, annotati il Seed Mnemonico e conservalo in un luogo sicuro!
-        <h2>Wallet Files Recovery Password</h2>
-        Make sure you write this password down. If you reset your device or uninstall the app, you
-        will need it to access your wallet again.<br/>
+        <h2>Password di ripristino dei file del portafoglio</h2>
+        Assicurati di aver salvato questa password. Se resetti il tuo dispositivo o disinstalli l'app
+        ne avrai bisogno per accedere di nuovo al tuo portafoglio.<br/>
         <h3>CrAzYpass</h3>
-        If the password displayed here is 52 alphanumeric characters in groups of 4 - congratulations!
-        Your wallet files are secured with a 256-bit key generated by your device&apos;s security
-        features based on the passphrase you chose (on creation or by changing it). This makes it
-        extremenly difficult to hack!<br/>
-        This feature is mandatory for all newly created wallets.
-        <h3>Legacy Password</h3>
-        If you see your passphrase here, your wallet files are not as secure as when using
-        a CrAzYpass. To fix this, simply select \"Change Passphrase\" from the menu. After entering
-        a new passphrase (maybe even the same one as before) the app will generate a CrAzYpass for
-        you and secure your wallet files with it. Write it down!
-        <h3>CrAzYpass wallets</h3>
-        If you ever need to reinstall Monerujo (for example after resetting your phone or switching
-        to a new one) or you want to use your wallet files on a different device or PC, you have to
-        use this Recovery Password in order to access your wallet again.<br/>
-        By selecting \"Change Passphrase\" from the menu, you can choose another passphrase. Beware
-        that this will generate a new Recovery Password. Write it down!
+        Se la password visualizzata qui contiene 52 caratteri alfanumerici in gruppi di 4 - Congratulazioni!
+        I file del tuo portafoglio sono criptati con una chiave a 256-bit generata dal tuo dispositivo
+        e basata sulla passphrase che hai scelto (alla creazione o quando l'hai modificata). Questo la rende estremamente
+        difficile da hackerare!<br/>
+        Questa feature è obbligatoria per tutti i nuovi portafogli.
+        <h3>Password precedente</h3>
+        Se vedi la tua passphrase qui, i file del tuo portaglio non sono così sicuri come 
+        quelli criptati da CrAzYpass. Per adeguare la tua password semplicemente seleziona \"Cambia Passphrase\" dal menu.
+        Dopo aver inserito una nuova passphrase (perfino la stessa usata in precedenza) l'applicazione genererà
```

Comment:
"perfino" ==> "eventualmente"

Reply:
@erciccione , this one? "Perfino" is "addirittura". Don't know if it matches the context here. I think "eventualmente" could be the right translation. What do you think about that?